readme.md

```diff
@@ -412,6 +415,19 @@ interface SubmitTerms {
 The extension MUST only send the request with active consent of the user.
 This is the first step where the user’s DID is revealed to the dApp.
 
+The previous message in the flow - `'submit-terms'` - contains the `claim` with an optional `owner` field containing a DID URI.
+This `owner` value being provided means that the attester is willing to issue the credential for this specific DID.
+Note that legacy attesters provide bogus DIDs in this field, this is likely the case if the extension does not control this DID.
```

Comment:
Well, we assume that the attester has a reasonable flow which will not tell the user "I’m making the credential for a DID you’ve never seen before". If they set the owner to a DID, they have likely confirmed that the user/extension controls this DID. The only anticipated exception is the case where the attester started with the legacy SDK which forced providing any DID value in the owner field, and they have been putting e.g. temporary light DID in it ever since.